src/main/webapp/i18n/en/multiRecordView.json

```diff
@@ -19,6 +19,7 @@
     "from": "From ",
     "lastUpdated": "Last partial update ",
     "lastCompleteReview": "Last complete review ",
+    "unknown": "unknown",
```

Comment:
I guess we can make it start with capital letter - "Unknown"

Reply:
This becomes a part of sentence - Last review unknown
should I still change the letter?

Reply:
ok I imagined it differently